### PR TITLE
Attempt to provision non-vm docker jupyterhub

### PR DESCRIPTION
**Pull request by manics**
A self-contained playbook for setting up JupyterHub on a physical host with a pre-existing LVM volume group. Also sets up cadvisor for container monitoring. `idr.connection` parameters are not configured by this playbook.

### CODE REVIEW

File: ansible/ome-analysis/ome-jupyter.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# In future if this is used to manage a cluster of Jupyterhub servers
+# some of the hardcode variables (e.g. volume sizes) should be moved into
+# group_vars
+
+- hosts: ome-jupyter
+
+  roles:
+
+  # Do we need this?
+  #- role: server-swap
```
Comment (joshmoore) [inline ansible/ome-analysis/ome-jupyter.yml]: What's this about?

Comment (joshmoore) [inline ansible/ome-analysis/ome-jupyter.yml]: What's this about?

File: ansible/ome-analysis/ome-jupyter.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    lvm_lvfilesystem: ext4
+    lvm_vgname: VolGroup00
+
+  - role: openmicroscopy.basedeps
+
+  - role: openmicroscopy.cli-utils
+
+  - role: openmicroscopy.versioncontrol-utils
+
+  # Ensure the system (especially kernel) is up to date
```
Comment (joshmoore) [inline ansible/ome-analysis/ome-jupyter.yml]: Does this mean: "make sure that `initial.yml` has been run?"

Comment (manics) [inline ansible/ome-analysis/ome-jupyter.yml]: Yes, and manually rebooted if necessary.

File: ansible/ome-analysis/ome-jupyter.yml (hunk 1 above)
Comment (manics) [inline ansible/ome-analysis/ome-jupyter.yml]: IIRC there was a problem with some centos 6 machines that unexpectedly used swap. I don't know if it's needed on centos 7. @kennethgillen ?

**Pull request merged**